pkg/network/node/ovscontroller.go

```diff
@@ -46,6 +48,41 @@ func (oc *ovsController) getVersionNote() string {
 	return fmt.Sprintf("%02X.%02X", oc.pluginId, ruleVersion)
 }
 
+func (oc *ovsController) updateOVSMetrics() {
+	flows, err := oc.ovs.DumpFlows()
+	if err == nil {
+		OVSFlows.Set(float64(len(flows)))
+	} else {
+		glog.Warningf("failed to dump OVS flows for metrics: %v", err)
```

Comment:
Always use utilruntime.HandleError when you eat errors.

Reply:
Fixed.